pytest_fixtures/component/taxonomy.py

```diff
@@ -180,22 +143,6 @@ def session_sca_manifest():
         yield manifest
 
 
-@pytest.fixture(scope='module')
-def module_entitlement_manifest():
```

Comment:
shows up in [tests/foreman/api/test_subscription.py](https://github.com/SatelliteQE/robottelo/blob/efabac69201ce9ddbf504ad50941ab4ed2e71341/tests/foreman/api/test_subscription.py#L71) and [tests/foreman/ui/test_contentview_old.py](https://github.com/SatelliteQE/robottelo/blob/efabac69201ce9ddbf504ad50941ab4ed2e71341/tests/foreman/ui/test_contentview_old.py#L234)